Makefile

```diff
@@ -15,6 +15,14 @@ RC_MUTABLE_IMAGE_NAME  = $(DOCKER_REPO)$(BASE_IMAGE_NAME):canary
 REL_IMAGE_NAME         = $(DOCKER_REPO)$(BASE_IMAGE_NAME):$(REL_VERSION)
 REL_MUTABLE_IMAGE_NAME = $(DOCKER_REPO)$(BASE_IMAGE_NAME):latest
 
+ifeq ($(REL_VERSION),)
+BROKER_VERSION="devel"
+else
+BROKER_VERSION=$(REL_VERSION)
+endif
+
+LDFLAGS = -w -X main.commit=$(COMMIT) -X main.commit=$(GIT_VERSION) -X main.version=$(BROKER_VERSION)
```

Comment:
`-X main.commit=$(COMMIT)` doesn't look right... it seems to write the same property as `-X main.commit=$(GIT_VERSION)`, and appears to use an undefined value. Guessing this was a copy/paste error.